web-components/src/components/cards/ProjectCard/ProjectCard.tsx

```diff
@@ -219,7 +219,10 @@ export function ProjectCard({
                     </Box>
                   </Box>
                   <OutlinedButton
-                    onClick={onButtonClick}
+                    onClick={event => {
+                      event.stopPropagation();
+                      onButtonClick && onButtonClick();
```

Comment:
I had to add `event.stopPropagation()` so the button click didn't also trigger a redirect (from clicking on the card as a whole).

`onButtonClick` is optional, but maybe it should just be required?

Reply:
Oh ok, I get it, it's because the Card itself is wrapped into a `<Link />`.
We should probably keep `onButtonClick ` optional just in case we need to use the card without the button.